packages/data-stores/src/site/actions.ts

```diff
@@ -314,7 +314,7 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 				apiNamespace: 'wpcom/v2',
 				body: {
```

Comment:
Would it be worth adding a TS type to `body` here to guarantee the data being sent to the endpoint? There's a lot of conditions in the following lines and that might help to keep bugs from slipping in here.

Reply:
Sure! I added `ThemeSetupOptions` interface to check the format of the body 👍